from core.exceptions import BaseError


class CoinApiError(BaseError):
    pass
